website/content/docs/commands/secrets/tune.mdx

```diff
@@ -57,8 +57,8 @@ flags](/docs/commands) included on all commands.
 
 - `-max-lease-ttl` `(duration: "")` - The maximum lease TTL for this secrets
   engine. If unspecified, this defaults to the Vault server's globally
-  configured maximum lease TTL, or a previously configured value for the secrets
-  engine.
+  configured [maximum lease TTL](/docs/configuration#max_lease_ttl), or a previously configured value for the secrets
+  engine. This value is allowed to override the system max TTL -- it can be longer or shorter.
```

Comment:
```suggestion
  engine. This value is allowed to override the server's global max TTL; it can be longer or shorter.
```

nit: using more similar wording to above just for maximum clarity.